### DISCUSSION TITLE
RSSI Wert

### DISCUSSION THREAD

**Discussion by frank-tanne** [hoylabs/OpenDTU-OnBattery#1349] [General]
Zum Verständnis ist -80 dBm eher schlecht oder gut

**Comment by Harisjems** [comment]
Ist mit Note ausreichend zum Vergleichen.
Je weniger, desto besser.
Mfg

**Reply by frank-tanne** [reply]
Das hatte ich mir schon gedacht. Mich wundert es nur das ich top empfang (1 Sekunde) ohne störung und andere haben Schwierigkeiten.

**Reply by Snusme83** [reply]
Servus, muss da nochmal nachhaken, da ich jetzt auch wieder Up-to-Date bin mit der oDTUoB!

@Harisjems: Du schreibst je weniger, desto besser. -80 dBm ist weniger als 0 dBm. Also wären -130 dBm noch besser, da noch weniger.

Habe aber gerade gelesen, dass je näher an 0 dBm, desto besser das Signal. Siehe: 
![Signalstärke](https://github.com/user-attachments/assets/cf00a89c-3d44-421f-853d-cd54d88350e1)

Wie kommt es zu der Aussage und hättest du vielleicht mal Material, wie dieser Wert sein kann und welcher Bereich sehr gut und welcher sehr schlecht ist. 

Danke im Voraus.

**Reply by schlimmchen** [reply]
Typische Verwechslung. Er meint den absoluten Wert der Zahl, du meinst den tatsächlichen. So wie es im Text steht, den du per Screenshot zitierst, stimmt es: Je näher an Null, desto besser.

**Comment by Harisjems** [comment]
So gesehen hast du recht.
War aber anderes rum gedacht.


Google sagt:-)

Was ist ein guter RSSI‑Wert?

Sehr guter Empfang	-1 dBm	-50 dBm

Guter Empfang	-51 dBm	-70 dBm

Mittelmäßiger Empfang	-71 dBm	-80 dBm

Ausreichender Empfang	-81 dBm	-90 dBm

Schlechter Empfang	-91 dBm	-105 dBm

Sehr schlechter Empfang	-106 dBm	höher

**Reply by Snusme83** [reply]
Ok, danke fürs Licht ins dunkle Bringen.

Hab nämlich auch -80dBm. Gibt es Maßnahmen, die man Treffen kann, um besseren Empfang herbeizuführen? Im Grunde passt es bei mir, aber hin und wieder dauert es scheinbar ein paar Sekunden länger, bis die DTU was empfängt!

**Reply by frank-tanne** [reply]
Besser Antenne, oder Entfernung verkürzen.

**Reply by Snusme83** [reply]
Entfernung ist 20 cm!
Bessere Antenne?
Hier noch eine Info zu RSSI:
Es wird bei HM- Wechselrichtern nur -80 dBm oder -30 dBm angezeigt!
![IMG_2850](https://github.com/user-attachments/assets/5d3d4ff4-37a3-470c-89df-94256353e62f)

**Reply by frank-tanne** [reply]
Bei 20 cm liegt es auf keinen fall an der Antenne. Vielleicht ist es auch zu nahe dran, das gibt es nämlich auch.

**Reply by Snusme83** [reply]
Zu nahe dran!🤔 Problem ist, dass die Pylontechs und die drei Ladregler und Display mit der oDTUoB verbunden sind und sich die ganze Peripherie in unmittelbarer Nähe befindet!🤷🏻‍♂️

**Reply by frank-tanne** [reply]
Was helfen kann die Antenne von WR abzuschirmen mit Alufolie oder so . Aber das müsste man mal testen.

**Reply by Snusme83** [reply]
Fraglich ist auch, ob das überhaupt optimierungsbedürftig ist:
![IMG_2853](https://github.com/user-attachments/assets/f5d120ad-1dad-450c-8db0-d8ee0cf2980e)

**Reply by SW-Niko** [reply]
Hallo,
bei mir sind es auch nur ca. 20cm. Ich habe die Sendeleistung auf minimal gestellt und die Antenne vom WR etwas abgeschirmt.
Wenn die beiden Geräte zu nahe beisammen sind dann kommt es anscheinend zu einer Übersteuerung an einem oder beiden Empfängern. Mit meinem Setup komme ich auf folgende Werte und das reicht für einen störungsfreien Betrieb.

![grafik](https://github.com/user-attachments/assets/f09a525c-9993-4dc7-b372-71a41c5dff44)

**Reply by Snusme83** [reply]
Ok, scheint bei dir besser zu klappen. 🤔
Handelt es sich bei dir um Nrf24 oder CMT2300, da der RSSI nicht zu sehen ist?

Sendeleistung hab ich auch auf minimal.

Welches Abfrageintervall hast du?

Wie sieht die Abschirmung deines WR aus?

**Reply by frank-tanne** [reply]
Als Abschirmung am WR kannst du einfach Alufolie um der Antenne wickeln. Bei mir ist grade das gegenteil 25 m und zwei Wände. Ich mach jede Sekunde eine abfrage und habe kaum Aussetzer.

**Reply by gitisgreat2023** [reply]
Was sind die TX re-requests? Etwa 50% von den Anfragen scheint mir viel...

**Reply by schlimmchen** [reply]
> Besser Antenne, oder Entfernung verkürzen.

Wie du schon selbst schriebst: Zu nah ist definitiv auch ein Problem. Jemand hat da auch mal eine Faustregel genannt, die die Wellenlänge einbezieht, leider kann ich die nicht zitieren. Jedenfalls ist 20cm zu nah dran.

> Fraglich ist auch, ob das überhaupt optimierungsbedürftig ist:

Naja, durchaus. Der Anteil teilweise empfangener Antworten ist schon arg hoch. Bei meinem HM-1500 sind es 2/15.500 in dieser Kategorie.

> Was sind die TX re-requests? Etwa 50% von den Anfragen scheint mir viel...

Ja, das ist sogar mehr als bei mir (5.200/15.500). Das passiert, wenn ein Paket nicht quittiert wurde -- so meine Interpretation. Im Detail kenne ich das Protokoll nicht. Dann wird ein Fragment/Paket nochmal übertragen. Das kostet Zeit und verlangsamt damit das Abfragen etwas. Ist also nicht ideal, aber eine Abfrage ist erst dann gescheitert, wenn kein RX Success gezählt wird.

**Reply by Snusme83** [reply]
@SW-Niko :  Ok, bei dir sieht es ja ganz gut aus. Was ist dein Geheimnis, dass der RX Success zu 100 % klappt?

Interessant wäre noch welche Antenne und welches Übertragungsintervall du eingestellt hast!

@frank-tanne : Hab die Antenne des Wechselrichters mit Alufolie umwickelt. Kann da keine Besserung feststellen. Habe immer noch -80dBm und hab jetzt mal von 1 Sekunden auf 2 Sekunden Abfrageintervall gestellt.

Die letzte Aktualisierung des Hoymiles HM-1500 sieht in etwa so aus:

0,1,2,0,1,2,0,1,2,3,0,1,2,0,1,2,3,4,0,1,2,0,1,0,1,2,0,1,2,0,1,2,3,4,5

also grundsätzlich 0,1,2, aber häufig halt auch mal 5 Sekunden, 6 Sekunden, 8 Sekunden,...

bei den Ladereglern (Victron SmartSolar 100/20) klappt es wunderbar mit der Aktualisierung:

0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1....

klar, die sind kabelgebunden, aber scheinbar geht es ja auch drahtlos zuverlässig!

Über Hilfe zu einer regelmäßigen Aktualisierung des Wechselrichters würde ich mich freuen.

Habe jetzt schon mehrere Lagen Alufolie um die Antenne des Wechselrichters gewickelt!🤷🏻‍♂️

**Reply by frank-tanne** [reply]
Die -80dBm kann ja nicht besser werden weil du sie ja mit Alu gedämpft hast. Und soll ja auch schlechter werden weil du zu nahe dran bist. Ich glaube auch nicht das du es besser hinbekommen wirst. Ab und zu mal 5 Sekunden ist ja nicht so schlecht. In der HF spielen viele Faktoren mit rein  aber das würde hier den rahmen sprengen.

**Reply by Snusme83** [reply]
Du hast ja gestern empfohlen, die Antenne des Wechselrichters in Alufolie einzuwickeln!🤔

Heute sollen die - 80dBm schlechter werden, weil ich zu Nahe dran bin!

Schlechter als -80dBm ist für mich -81dBm, -82dBm,...

Besser als -80 dBm ist für mich -79dBm, -78dBm,...

Abgesehen davon, dass in der oDTUoB bei Hoymiles Wechselrichtern lediglich -80 dBm und -30 dBm angezeigt werden, sollte ja mein Ziel sein, dass in der oDTUoB -30dBm (da besser) angezeigt wird, damit die Übertragung einwandfrei funktioniert.

So wie es sich für mich darstellt, klappen 77% der Übertragungen einwandfrei und 23 % eben nicht.

Dachte durch das empfohlene Anbringen der Alufolie erhöhe ich faktisch die Distanz zwischen oDTUoB und Wechselrichter!

**Reply by schlimmchen** [reply]
![image](https://github.com/user-attachments/assets/4bd99427-f6ac-4234-a0cd-0dbf1ed7c4ee)

Also bei mir schwankt es gerade zwischen -30 dBm und -80 dBm, bin also scheinbar auf der Schwelle bei -64 dBm. 12.700 Re-Requests bei 33.000 Anfragen.

**Reply by Snusme83** [reply]
Bei mir 125282 Re-Requests bei 38557 Anfragen. Was hat das zu bedeuten?

**Reply by frank-tanne** [reply]
Mit der Alufolie erhöhst du auch die Distanz! Eine Antenne hat immer eine Dämpfung. Wenn es sehr gut wäre hättest -0dBm, Schlecht wäre dann -100dBm . Wenn deine Zwei Antennen direkten Sichtkontakt hätten sie -0dBm .Eine Wand dazwischen oder die Alufolie vielleicht noch -80dBm. Wenn du da mehr wissen willst dann lies mal Rothammels Antennenbuch.

**Reply by Snusme83** [reply]
Meine beiden Antennen haben direkten Sichtkontakt, da nur 20 cm voneinander entfernt!!!

**Comment by Snusme83** [comment]
![IMG_2859](https://github.com/user-attachments/assets/4ab7be2a-60ee-45fe-af54-04b92414befa)

**Reply by SW-Niko** [reply]
>interessant wäre noch welche Antenne und welches Übertragungsintervall du eingestellt hast!
Ich benutze einen ESP32-S3 Kit mit auf das Layout aufgedruckter Antenne (ohne externer Antenne.) Abfrageintervall 4 Sek.

Du kannst jetzt noch versuchen den Winkel zwischen den beiden Antennen zu verändern und ausprobieren was passiert wenn du mehr Abstand machst.

>Dachte durch das empfohlene Anbringen der Alufolie erhöhe ich faktisch die Distanz zwischen oDTUoB und Wechselrichter!

Ja, kann man so sagen. Genaugenommen wird die Signalstärke gedämpft
PS: Externen Antenne am ESP32 und 20cm Abstand ist keine gute Kombination.

**Reply by Snusme83** [reply]
Winkel zwischen den Antennen kann ich probieren.
Mehr Abstand ist schwierig, da wie bereits erwähnt 3 Laderegler an ihrer Position, und Pylontechs ebenso.

PS:  "Externen Antenne am ESP32 und 20cm Abstand ist keine gute Kombination." - Was ist dann die Lösung, außer mehr Distanz, was ja mit einem Umbau bzw. längeren Kabeln verbunden wäre?

**Reply by frank-tanne** [reply]
Vielleicht reicht es ja den WR zu drehen das die Antenne in die andere richtung zeigt. Bei den Frequenzen machen 20cm viel aus.

**Comment by Tomv888** [comment]
Ich habe das Steckernetzteil in etwa 20cm entfernt von der DTU.Das USB-Kabel zwischen Netzteil und DTU war 1,5m lang und aufgerollt hinter der DTU versteckt.Hier hatte ich einen RSSI zwischen 75 und 80 dBm.

Aus optischen Gründen habe ich das USB-Kabel dann auf 25cm länge eingekürzt.
Seitdem liegt der RSSI zwischen 56 und 63 dBm.Ich vermute,das ich durch nun weniger Schirmgeflecht und GND Leitung
einer besser abgeglichene Antenne habe (Stehwelle)

**Reply by Snusme83** [reply]
Ich kann machen, was ich will, immer das selbe!
Welche Antenne hast du und woher weißt du den RSSI- Wert so genau? So genau wird der ja gar nicht angezeigt von der oDTUoB!🤔

**Reply by frank-tanne** [reply]
Er meint bestimmt den RSSI-Wert von Wlan . Du hast Recht vom nRf24 wird er überhaupt nicht so genau angezeigt

**Comment by Tomv888** [comment]
Nein,nicht vom WLAN,von der DTU zum HMS.Der Wert wird in der Konsole angezeigt,oder neuerdings auch im Hauptbild.

**Reply by frank-tanne** [reply]
Hms haben wir nicht das ist ein anderes Funkmodul (CMT2300A) wir haben nRf24.

**Reply by Snusme83** [reply]
Ok, dann wird es vermutlich so sein, dass bei dir Info (RSSI des zuletzt empfangenen Paketes) in der Funkstatistik mit HM- Invertern nicht alle Hoymiles Wechselrichter gemeint sind, sondern lediglich die HM-Serie, welche mit NRF24 kommunizieren --> -30 dBm und -80 dBm.

Die Hoymiles- Serien, welche mit CMT2300A kommunizieren können die RSSI- Werte scheinbar detailierter anzeigen.

@SW-Niko: Kann es sein, dass du als Antenne auch ein CMT2300A aufgrund eines HMS- oder HMT- Wechselrichters hast? (Deswegen habe ich auch schon zweimal zuvor nach der Art der Antenne gefragt)! Denke, dass ein Vergleich von NRF24 und CMT2300A dann wenig Sinn macht - trotz alledem betreibt aber:

@schlimmchen einen HM-1500 (wie ich). Bei ihm wird scheinbar nur -80 dBm und -30dBm im Wechsel angezeigt. 12700 Re-Requests bei 33000 Anfragen würde dann auch bedeuten, dass etwa 1/3 nicht funktioniert. Dementsprechend müsste deine Funktstatistik ähnlich aussehen wie bei mir, mit dem Unterschied, dass RSSI- Wert mal -80 dBm und mal -30 dBm angezeigt wird.

Habe jetzt eine zweite oDTUoB erstellt mit angeschlossenem NRF24-Modul, egal wie weit ich weg bin vom Wechselrichter, etc. --> immer -80 dBm!

Mein NRF 24 ist von Amazon: Hailege NRF24L01+ 2.4 Wireless Transceiver Module....

**Reply by Snusme83** [reply]
@frank-tanne: Da du scheinbar auch ein NRF24 Modul hast, kannst du bitte mal ein Bild von deiner Funkstatistik senden?

**Reply by Snusme83** [reply]
Hier nochmal eine aktuelle von mir:
![Funkstatistik](https://github.com/user-attachments/assets/9cf36a52-8c86-443e-b967-f184adfb5ed9)

**Reply by SW-Niko** [reply]
> Kann es sein, dass du als Antenne auch ein CMT2300A aufgrund eines HMS- oder HMT- Wechselrichters hast?

Nein ich habe einem HM und ein NRF24L01+ ohne externe Antenne.
![grafik](https://github.com/user-attachments/assets/eb4bca7e-d3c2-47f5-a97a-403b1bb39b32)

Hast du schon mal im Upstream Project nach Informationen gesucht? Die haben die Kommunikation mit den HM auch implementiert. Vielleicht findest du da eher einen Funkprofi, der dir bei deinem Problem helfen kann.

**Reply by Snusme83** [reply]
Kann ich mal machen. Wieso wird bei deiner Radio Statistics der dBm Wert nicht angezeigt?

**Reply by SW-Niko** [reply]
>Wieso wird bei deiner Radio Statistics der dBm Wert nicht angezeigt?

Ich habe aktuell noch eine ältere Entwicklungs-Version auf meinem Produktiv-System. Werde ich den nächsten Tagen wieder auf Stand bringen.

**Reply by Snusme83** [reply]
Danke für die Rückantwort. Wegen mir muss das aber nicht sein!😉
Frage: Bei der Aktualisierung deines Wechselrichters erfolgt dies wahrscheinlich genau in dem Intervall, das du eingestellt hast, oder?

**Comment by Harisjems** [comment]
Ich hab beide WR HM Serie und HMS
HMS ca 3 m entfernt 
HM 20M entfernt, alles läuft über Fusion V2 Board und klebeantennen.

![Screenshot_20241101_144327_Samsung Internet](https://github.com/user-attachments/assets/1a973191-dfff-48fc-8fe2-f1151456f178)

![Screenshot_20241101_144311_Samsung Internet](https://github.com/user-attachments/assets/0e8f70cc-a14a-4159-860c-fe4c65639813)
Bild 1 HM
Bild 2 HMS

**Reply by Snusme83** [reply]
Danke Harisjems. Was hast du für ein Intervall bei den wenigen Anfragen?

Du hast auch -80 dBm, aber bei dir Empfang erfolgreich 100%. Bei mir lediglich um die 70%.🤷🏻‍♂️

**Comment by Harisjems** [comment]
Das Intervall beträgt 3 Sekunden.
OpenDTU habe ich vor ca. 2 Stunden wegen Bastelarbeiten neu gestartet.
Verlorene Anfragen/Pakete sind aber auch bei längerer Laufzeit hier nicht problematisch.
Mfg

**Reply by Snusme83** [reply]
Was meinst du mit verlorenen Paketen?

**Reply by frank-tanne** [reply]
Kannst du dein Display abziehen? Wenn ja zieh mal +3v ab und schau dir dann mal den empfang an

**Comment by Harisjems** [comment]
Empfangfehler
Beschädigt empfangen
Ich würde es als verloren definieren :-)

Dass ich davon ausgehen kann, dass paar Störungen auf dieser Frequenz zu erwarten sind: Nachbarn, Smarthome, Starkink, CB Funk,Nato Base 5 km weiter usw.:-)

**Comment by frank-tanne** [comment]
So sieht es bei mir aus 20m zwei Wände dazwischen,Abfrage 1sek
![Screenshot 2024-11-01 152400](https://github.com/user-attachments/assets/529e4ff2-7efe-4db3-b3ee-9f7cf2fd819f)

**Reply by Snusme83** [reply]
Beschädigt Empfangen ist ja bei mir auch 0 %, aber wieso zum Teufel sind die teilweise Empfangen so viele?
Ich denke, das wird dann immer der Fall sein, wenn das eingestellte Intervall nicht eingehalten wird!

Hab die Abfrage von 1 Sek. auf  2 Sek. erhöht und trotzdem komme ich nicht annähernd an Empfang erfolgreich 100 %!

**Comment by Harisjems** [comment]
Wozu soll das 1-Sekunden-Intervall gut sein?
Ich bilde mir ein, dass 3 Sekunden zu viel sind.

**Reply by Snusme83** [reply]
Ich Bild mir ein, dass die Regelung genauer wird!

Angenommen Fön an. Innerhalb weniger Sekunden unterstützt der Wechselrichter mit Strom!

Ist das Intervall auf z. B.: 1 Minute ist der Fön 1 Minute an und erst dann hilft der Wechselrichter mit. 

Fön aus - bedeutet dann auch, dass der Wechselrichter 1 Minute Strom verschenkt! Weißt was ich meine?

**Reply by frank-tanne** [reply]
Kannst du dein Display abziehen? Wenn ja zieh mal +3v ab und schau dir dann mal den empfang an

**Reply by schlimmchen** [reply]
> Ich bilde mir ein, dass 3 Sekunden zu viel sind.

Für die Regelung mit dem DPL ist weniger besser. AhoyDTU hat, soweit ich das im discord mitgelesen habe, inzwischen die Pausen zwischen den Abfragen ganz aufgegeben, was mir sinnvoll erscheint.

> Weißt was ich meine?

Ja, du hast recht. Im Detail bin ich mir nicht sicher, weil ich glaube mich zu erinnern dass auch das Abarbeiten der Befehle an den Inverter von diesem Intervall abhängen, was die Situation verschärft, also zusätzlich verlangsamt.

**Reply by Snusme83** [reply]
> Kannst du dein Display abziehen? Wenn ja zieh mal +3v ab und schau dir dann mal den empfang an

Ok, ich glaub ich weiß auf was du raus willst! Display weg, damit der NRF24 mehr Saft bekommt, oder? Display ist jetzt aus.

Denke aber nicht, dass der NRF24 zu wenig Saft bekommt, da der ESP32-S3 vom Breakoutboard mit Strom versorgt wird, aber Versuch macht ...!

Display bleibt jetzt mal aus.

**Reply by frank-tanne** [reply]
Genau das meine ich, das proplem hatte ich nämlich wo ich auf ESP32-S3 umgestellt habe. Bei mir kam fast nichts mehr an

**Comment by Harisjems** [comment]
O. K.
Ich war bisher der Meinung, dass die Abfrage allein nicht maßgeblich ist, sondern die Reaktion des WR auf das Kommando bzw. die Ausführung des Kommandos.
Bei einem Abfrageintervall von drei Sekunden gewinnt man gegenüber einem Intervall von fünf Sekunden zwei Sekunden.
Die interne Regelung des WR allein beeinflusst das weniger, der braucht seine fünf bis zehn Sekunden.
Wenn der WR in einer Stunde 100-mal Regeln muss, ist es effizienter von Logik als 300-mal Regeln.

Oder war ich die letzten zwei Jahre auf dem Holzweg, wer weiß? :-)

**Reply by Snusme83** [reply]
Kurios, 3 V vom Display abgezogen:

- Display logischerweise aus
- Gesamtertrag Mppt weg
- Ertrag heute weg
- Gesamtleistung weg
- Wechselrichter wird nicht mehr aktualisiert und steht auf dem Limit, das vorm Abziehen war.😵‍💫

**Reply by frank-tanne** [reply]
zieh mal die andern 3 auch noch ab

**Reply by frank-tanne** [reply]
Bei mir ist jetzt gleich Schluss kommen nur noch 3W

**Comment by Snusme83** [comment]
Alle 4 abgezogen. Jetzt läuft alles, bis aufs Display.

Und jetzt - Abwarten und Tee trinken?😎

Habe noch sagenhafte 6/7 Watt bei 6 Modulen a 370 Wp!💪🏻

**Reply by frank-tanne** [reply]
jetzt bin ich mal neugierig.

**Reply by Snusme83** [reply]
Kann dann erst morgen was sagen, aber die Regelung/ Aktualisierung ist wieder nicht im 2-Sekunden-Takt!😳😕🤨

**Reply by frank-tanne** [reply]
Wenn es besser ist ,melde dich noch mal da habe ich eine lösung. Wenn nicht weiß ich auch nicht weiter. Und die nRf24 Sendeleistung auf 0dBm stellen

**Reply by Snusme83** [reply]
![IMG_2877](https://github.com/user-attachments/assets/7fd883f7-a5b6-469e-bc13-0308f79c47f1)


🤷🏻‍♂️

**Reply by frank-tanne** [reply]
Den Elko hast du auch verbaut? Bei mir sah es jetzt noch schlechter aus. Ich habe die Statistik zurückgesetzt jetzt ist alles in Ordnung.

**Reply by frank-tanne** [reply]
![Screenshot 2024-11-02 095617](https://github.com/user-attachments/assets/4bc44cc5-9cf6-471e-ace2-692d49087721)

**Reply by Snusme83** [reply]
Elko hab ich keinen verbaut. Bei dir sah es jetzt noch schlechter aus als bei mir? Hast du nen Elko verbaut?

**Reply by frank-tanne** [reply]
![Screenshot 2024-11-02 100104](https://github.com/user-attachments/assets/e70852f4-acc7-4ef0-9e45-f646da111053)
Das war vorher . Elkos habe am Display und am nRf24

**Reply by frank-tanne** [reply]
Meine Vermutung, wenn der WR hochfährt aber noch zuwenig produziert wird sind die wehrte schlecht.

**Reply by Snusme83** [reply]
Mein WR ist aufgrund des aktuellen Wetters lediglich im Standby!

**Reply by frank-tanne** [reply]
Ich habe dein Foto von deinem Aufbau in einem anderen Beitrag gesehen (ESP32 Stromversorgung über Pylontech) . Deine zwei Step-Down Wandler machen natürlich auch HF Störung, Über die Luft (Antenne) aber auch Kabel.

**Reply by Snusme83** [reply]
Ok, ist zwar nur einer, aber dann werde ich die oDTUoB wieder ganz normal mit Strom versorgen.

**Reply by frank-tanne** [reply]
Einer der vom Akku kommt und einer auf deinem Bord. Ich weiß nicht ob das eine Rolle spielt da habe ich zu wenig ahnung. Ich weiß nur das die Step-Down Wandler Störungen erzeugen können . Das nRf Modul hat ja auch keinerlei abschirmung . Deswegen habe ich auch das Modul 
![Screenshot 2024-11-02 131833](https://github.com/user-attachments/assets/9f1b07c8-b982-467c-9a1e-615065192bbf)

**Reply by frank-tanne** [reply]
Möglichkeit wäre den Grünen Step-Down direkt am Akku platzieren und dan mit eine langen Kabel direkt auf +5v und GND am ESP32 anschließen.
Der Step-Down muss natürlich dann genau auf 5V eingestellt werden. Und dann brauchst du den auf den Bord überhaupt nicht mehr.

**Comment by Harisjems** [comment]
Kann ich toppen?
Bei 5 kWp und 12 Modulen kann ich mich heute zu den Kapitalisten zählen. :-)

![Screenshot_20241101_165317_Samsung Internet](https://github.com/user-attachments/assets/4a088d8d-f019-4272-b354-1796c01e58ae)

**Reply by Snusme83** [reply]
Da kann man aber Nix erkennen!🤷🏻‍♂️

So sah es bei uns heute aus:
![IMG_2872](https://github.com/user-attachments/assets/f2408a78-5f67-4bb5-a7d2-3cbdd5b5a938)
